app/helpers/form_helper.rb

```diff
@@ -316,8 +316,8 @@ def add_label options, f, attr
     label_size = options.delete(:label_size) || "col-md-2"
     required_mark = check_required(options, f, attr)
     label = ''.html_safe + options.delete(:label)
-    if label.empty? && f.try(:object) && ((clazz = f.object.class).respond_to?(:gettext_translation_for_attribute_name))
-      label = s_(clazz.gettext_translation_for_attribute_name attr).html_safe
```

Comment:
Why are you removing the `gettext_translation_for_attribute_name`? This is used to provide context for translations. e.g. - `Template.gettext_translation_for_attribute_name(:name) => "Template|Name"`, which is then parsed by gettext to provide the correct translation for the context (since it is possible that attributes that have the same name in one language might have different names in another depending on the context)